website/docs/d/tg_connection_prefix_filter.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_connection_prefix_filter"
+description: |-
+  Manages IBM Cloud Infrastructure Transit Gateway Connection Prefix Filter.
+---
+
+# ibm_tg_connection_prefix_filter
+Retrieve information of an existing IBM Cloud infrastructure transit gateway connection prefix filter as a read only data source. For more information about Transit Gateway Prefix filters, see [adding and deleting prefix filters](https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-adding-prefix-filters&interface=ui).
+
+## Example usage
+
+```terraform
+data "ibm_tg_connection_prefix_filter" "tg_prefix_filter" {
+    gateway = ibm_tg_gateway.new_tg_gw.id
+    connection_id = ibm_tg_connection.test_ibm_tg_connection.connection_id
+	filter_id = ibm_tg_connection_prefix_filter.test_tg_prefix_filter.filter_id
+}
+```
+
+## Argument reference
+Review the argument references that you can specify for your data source. 
+
+- `gateway` - (Required, String) The unique identifier of the gateway.
+- `connection_id` - (Required, String) The unique identifier of the gateway connection
+- `filter_id` - - (Required, String) The unique identifier of the gateway connection prefix filter
+
+## Attribute reference
+In addition to the argument reference list, you can access the following attribute references after your data source is created. 
+
+- `created_at` - (Timestamp) The date and time resource is created.
```

Comment:
(Timestamp) ==> (String)
It should be String, there is no timestamp type in terraform